src/commands/utility/info/emote.js

```diff
@@ -0,0 +1,35 @@
+const { Command, SwitchbladeEmbed, CommandError } = require('../../../')
+const moment = require('moment')
+
+module.exports = class InfoEmoji extends Command {
+  constructor (client) {
+    super({
+      name: 'emoji',
+      parent: 'info',
+      requirements: { guildOnly: true, botPermissions: ['MANAGE_EMOJIS'] },
+      parameters: [{
+        type: 'emoji',
+        missingError: 'commands:info.subcommands.emote.missingEmoji'
+      }]
+    }, client)
+  }
+
+  async run ({ t, author, channel, language }, emoji) {
+    moment.locale(language)
+    try {
+      const emojiAuthor = await emoji.fetchAuthor()
+      const embed = new SwitchbladeEmbed(author)
+        .setTitle(`${emoji.animated ? `${t('commands:info.subcommands.emoji.animatedTag')} ` : ''}${emoji.name}`)
+        .setDescription(`
+        **${t('commands:guildinfo.createdAt')}**: ${moment(emoji.createdAt).format('LLL')}\n(${moment(emoji.createdAt).fromNow()})
+        **ID**: \`${emoji.id}\` \`\`<:${emoji.name}:${emoji.id}>\`\`
```

Comment:
```suggestion
        **ID**: \`${emoji.id}\` \`\`${emoji.toString()}\`\`
```
Animated emojis don't display the tag correctly. and by using `emoji.toString()`, it'll return the right format for that emoji, regardless of it being animated or not.